### PR TITLE
Add ignore test failures regex option

### PR DESCRIPTION
**Pull request by lukaville**

### CODE REVIEW

File: core/src/test/kotlin/com/malinskiy/marathon/execution/TestLogsProvider.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.malinskiy.marathon.execution
+
+import com.malinskiy.marathon.report.logs.BatchLogs
+import com.malinskiy.marathon.report.logs.LogReport
+import com.malinskiy.marathon.report.logs.LogsProvider
+
+class TestLogsProvider(private val logs: Map<String, BatchLogs>) : LogsProvider {
+
+    override fun getFullReport(): LogReport {
+        return LogReport(logs)
```
Comment (idyatlov) [inline core/src/test/kotlin/com/malinskiy/marathon/execution/TestLogsProvider.kt]: = LogReport(logs)

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/logcat/parse/LogcatEventsAdapter.kt [hunk 2]
```diff
@@ -40,6 +40,10 @@ class LogcatEventsAdapter(private val parsedEventsListener: LogcatEventsListener
 
     private fun LogcatMessage.parseZygoteExitReason(device: AndroidDevice): FatalError? {
         if (tag == ZYGOTE_TAG) {
+            if (body.contains(ZYGOTE_KILLED_EXTERNALLY_EXIT_REASON)) {
```
Comment (idyatlov) [inline vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/logcat/parse/LogcatEventsAdapter.kt:43]: Maybe add test to check that null will be returned? Or you can skip it if it will take a lot of time
